src/main/kotlin/persistence/DirectoryProvider.kt

```diff
@@ -47,4 +48,21 @@ class DirectoryProvider(private val appName: String) : IDirectoryProvider {
         file.mkdirs()
         return file
     }
+
+    // Creates a user subdirectory and stores their hash info
+    fun getAudioDirectory(): File {
+        return createInternalAppDirectory("Audio")
```

Comment:
The directory should probably be something more like `getAppDataDirectory("users${separator}audio")` for better organization